benchmarks/e2e/config/config.go

```diff
@@ -29,3 +29,15 @@ func (c *BenchmarkConfig) GetValidTenant() (TenantSpec, error) {
 
 	return c.TenantB, nil
 }
+
+func (c *BenchmarkConfig) ValidateTenant(t TenantSpec) (error) {
+	if c == nil {
+		return errors.New("Please fill in a valid/non-empty config.yaml")
+	}
+
+	if !reflect.DeepEqual(t, TenantSpec{}) {
+		return nil
```

Comment:
shouldn't this return an error?

Reply:
It was confusing to me so i tested it and it was working well, no problem i will run again the test and get back to you

Reply:
I am agreed that it should give error but its working according to required behavior only when we compliment it by `not !` in front of it. Actually if we see https://github.com/kubernetes-sigs/multi-tenancy/blob/a99b6dbd22693bf23f119226184df87301549f32/benchmarks/e2e/config/config.go#L26 this is also behaving exact like this! Lets say i remove `not !` from  front of it and fill the Tenant A and comment out the Tenant B in `config.yaml` then it shows error.